### PR TITLE
Update BCD info, part 38

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Adding to #19185 

The PR updates BCD info in WebAPI docs.

This covers files that have only `experimental` header modifications.

### CODE REVIEW

File: files/en-us/web/api/mouseevent/layerx/index.md [hunk 1]
```diff
@@ -12,7 +12,6 @@ tags:
   - Non-standard
 browser-compat: api.MouseEvent.layerX
 ---
-
```
Comment (teoli2003) [inline files/en-us/web/api/mouseevent/layerx/index.md:15]: That's fine here, but avoid removing them: Prettier will re-add them…

File: files/en-us/web/api/xrwebgllayer/index.md [hunk 2]
```diff
@@ -6,7 +6,6 @@ tags:
   - API
   - AR
   - Augmented Reality
-  - Experimental
```
Comment (teoli2003) [inline files/en-us/web/api/xrwebgllayer/index.md:9]: Don't remove this experimental, it should be. Only 1 engine supports it.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/mouseevent/layerx/index.md (hunk 1 above)
Comment (OnkarRuikar) [reply files/en-us/web/api/mouseevent/layerx/index.md:15]: Most of the files in the repo do not have a blank line after front matter. So when prettier will run it'll add a blank line after front matter?

File: files/en-us/web/api/xrwebgllayer/index.md (hunk 2 above)
Comment (OnkarRuikar) [reply files/en-us/web/api/xrwebgllayer/index.md:9]: But the [BCD table](https://developer.mozilla.org/en-US/docs/Web/API/XRWebGLLayer#browser_compatibility) isn't showing experimental. Also there is no experimental banner on the page. Adding exceptions for individual files in the script will be hard.

The PR series [Mark features with one engine support 'experimental'](https://github.com/mdn/browser-compat-data/pull/17391) hasn't landed yet in BCD repo. Do you want me to create a separate PRs for this in `mdn/browser-compat-data`?

Comment (teoli2003) [reply files/en-us/web/api/xrwebgllayer/index.md:9]: Yes, please, this is a very strange case. All the other WebXR feature are marked experimental on bcd.

Comment (OnkarRuikar) [reply files/en-us/web/api/xrwebgllayer/index.md:9]: Here is the PR
- https://github.com/mdn/browser-compat-data/pull/17535

Comment (teoli2003) [reply files/en-us/web/api/xrwebgllayer/index.md:9]: Merged.

File: files/en-us/web/api/xrwebgllayer/index.md [hunk 3]
```diff
@@ -6,7 +6,6 @@ tags:
   - API
   - AR
   - Augmented Reality
```
Comment (teoli2003) [inline files/en-us/web/api/xrwebgllayer/index.md:8]:
```suggestion
  - Augmented Reality
  - Experimental
```

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**